modules/rosapps/applications/explorer-old/explorer.rc

```diff
@@ -178,6 +178,9 @@ CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST "explorer.exe.manifest"
 #ifdef LANGUAGE_SV_SE
     #include "explorer-sv.rc"
 #endif
+#ifdef LANGUAGE_TR_TR
```

Comment:
Not sure about the necessity of translating the "old" explorer, since it's deprecated.